### PR TITLE
Update github issue templates

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#### Steps to reproduce
+
+  1. ...
+
+#### Code example
+
+```ruby
+# example
+```
+
```
Comment (quartzmo) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Can we add something like:
```
#### Backtrace / expected output

```sh
$ irb
irb(main):001:0> foo
NameError: undefined local variable or method `foo' for main:Object
	from (irb):1
	from .rbenv/versions/2.4.2/bin/irb:11:in `<main>'
```
```

Comment (JustinBeckwith) [reply .github/ISSUE_TEMPLATE/bug_report.md]: 👋 Feel free to submit follow up PRs that mold this to your liking :) The important part for me was the consistent language around Support requests, contracts, etc.

**Comment by quartzmo** [comment]
@JustinBeckwith Why does this file move? `CONTRIBUTING.md → .github/CONTRIBUTING.md`

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (quartzmo) [reply .github/ISSUE_TEMPLATE/bug_report.md]: kk, will do

**Comment by JustinBeckwith** [comment]
I suppose I didn't have to, but I mostly did this for consistency.  GitHub will show this on the sidebar for PRs if it's in the root or `.github` directory:
https://help.github.com/articles/setting-guidelines-for-repository-contributors/

Having it in the same place for every repo we manage will make my life a tad easier :)

**Pull request merged**